src/pages/saved-exercises.js

```diff
@@ -0,0 +1,63 @@
+import {nanoid} from 'nanoid';
+import Router from 'next/router';
+import {Fragment} from 'react';
+
+import DeleteIcon from '../components/Icons/DeleteIcon';
+import MoveBackButtonButton from '../components/MoveBackButton';
+import SavedTimerCard from '../components/SavedTimerCards';
+import StyledButton from '../components/StyledButtons';
+import {StyledHeader, StyledH1} from '../components/StyledHeader';
+import StyledLayout from '../components/StyledLayout';
+import StyledSavedExerciseContainer from '../components/StyledSavedExerciseContainer';
+import useSavedExercises from '../hooks/useSavedExercises';
+import useStore from '../hooks/useStore';
+
+export default function SavedExercises() {
+	const savedExercises = useSavedExercises(state => state.savedExercises);
+	const loadSaveExercise = useStore(state => state.loadSaveExercise);
+	const deleteExcercise = useSavedExercises(state => state.deleteExcercise);
+
+	return (
+		<>
+			<StyledLayout>
+				<StyledHeader>
+					<MoveBackButtonButton />
+					<StyledH1>Saved Exercises</StyledH1>
+				</StyledHeader>
+				{savedExercises.map(savedE => (
```

Comment:
`savedExercise`  = singular,  would be better than `savedE`

Reply:
totally right thank you for the hint